### DISCUSSION TITLE
着地タイミング修正が行われた場合のdouble_support_ratio_after

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#53] [Q&A] [answered]
着地タイミング修正が行われた場合には、
1. `修正前のstep_time` * `double_support_ratio_after`
2. `修正後のstep_time` * `double_support_ratio_after`

どちらが使用されるのでしょうか。

多くの箇所では2が使用されていますが、以下の箇所では1が使用されているように見えます。

https://github.com/YutaKojio/hrpsys-base/blob/a1e7119160a42bdc26c8b44c082981220c5b8abc/rtc/AutoBalancer/GaitGenerator.h#L454-L459
https://github.com/YutaKojio/hrpsys-base/blob/a1e7119160a42bdc26c8b44c082981220c5b8abc/rtc/AutoBalancer/GaitGenerator.cpp#L325-L328

**Comment by YutaKojio** [comment, marked as answer]
さすがよく見てるね！
現在は2.にしているつもりでしたが，昔1.にしていたときのが直しきれてませんでした．
https://github.com/YutaKojio/hrpsys-base/commit/507d06f23286cc062598ce6a65b2862ce0ba63d0
で修正したつもりです．

シミュレーションで歩行中に歩行時間を 1s -> 5s や 10s -> 5s に変えてみましたが，歩行の挙動としては変更前後でほとんど違いなさそうでした．

**Reply by YutaKojio** [reply]
さっき追加したコメントが嘘だったので消しました https://github.com/YutaKojio/hrpsys-base/commit/bacb4a0a2c099ae0e4fda7aa4f696fbf289e4111
